modules/virt-about-changing-removing-mediated-devices.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+
+[id="about-changing-removing-mediated-devices_{context}"]
+= About changing and removing mediated devices
+
+{VirtProductName} updates the cluster's mediated device configuration if:
```

Comment:
Can this be rephrased to avoid the possessive? Also need to finish the list introduction sentence.